kernel/patches/4.19.x/0001-enable-ptp_kvm-for-arm64-on-4.19.patch

```diff
@@ -0,0 +1,421 @@
+From e9a740c5cf801acd4446325e45d90b54b0ce5e6d Mon Sep 17 00:00:00 2001
```

Comment:
@jongwu Have you submitted this patch upstream? If so, can you include a link in the commit message?
Does the host kernel need to be patched as well to support the hypercall?

Reply:
@amshinde yeah, I have pushed this patch to upstream. but my patch base on v5.3, so it can't be patched into v4.19. this commit is largely different with the one in upstream.
To enable ptp_kvm on arm64, a patch needed for host, but I don't know where can I put it.
the patch in upstream can be a hint. I will link it in commit message.